frontend/packages/dev-console/src/components/add/AddPageLayout.tsx

```diff
@@ -20,49 +19,46 @@ const AddPageLayout: React.FC<AddPageLayoutProps> = ({ title, hintBlock }) => {
   const [activeNamespace] = useActiveNamespace();
   const addActionGroupExtensions = useExtensions<AddActionGroup>(isAddActionGroup);
   const [addActionExtensions, resolved] = useAddActionExtensions();
-
   const [showDetails, setShowDetails] = useShowAddCardItemDetails();
 
-  if (!resolved) {
-    return <LoadingBox />;
-  }
-
-  const defaultHintText = t(
-    'devconsole~Select a way to create an Application, component or service from one of the options.',
-  );
-
-  const defaultHintBlock: React.ReactNode = (
-    <div className="odc-add-page-layout-hint-block">
-      <div className="odc-add-page-layout-hint-block__text">{defaultHintText}</div>
-      <div className="odc-add-page-layout-hint-block__details-switch">
-        <Tooltip content={t('devconsole~Show or hide details about each item')} position="top">
-          <Switch
-            aria-label={
-              showDetails
-                ? t('devconsole~Show add card details')
-                : t('devconsole~Hide add card details')
-            }
-            isChecked={showDetails}
-            onChange={(checked) => {
-              setShowDetails(checked);
-            }}
-            data-test-id="odc-add-page-details-switch"
-          />
-        </Tooltip>
-        <span className="odc-add-page-layout-hint-block__details-switch__text">
-          {showDetails ? t('devconsole~Details on') : t('devconsole~Details off')}
-        </span>
+  const getHintBlock = (): React.ReactNode => {
+    const defaultHintText = t(
+      'devconsole~Select a way to create an Application, component or service from one of the options.',
+    );
+    return (
+      <div className="odc-add-page-layout-hint-block">
+        <div className="odc-add-page-layout-hint-block__text">{defaultHintText}</div>
+        <div className="odc-add-page-layout-hint-block__details-switch">
+          <Tooltip content={t('devconsole~Show or hide details about each item')} position="top">
+            <Switch
+              aria-label={
+                showDetails
+                  ? t('devconsole~Show add card details')
+                  : t('devconsole~Hide add card details')
+              }
+              isChecked={showDetails}
+              onChange={(checked) => {
+                setShowDetails(checked);
+              }}
+              data-test-id="odc-add-page-details-switch"
+            />
+          </Tooltip>
+          <span className="odc-add-page-layout-hint-block__details-switch__text">
+            {showDetails ? t('devconsole~Details on') : t('devconsole~Details off')}
+          </span>
+        </div>
       </div>
-    </div>
-  );
+    );
+  };
 
   return (
-    <PageLayout title={title} hint={hintBlock || defaultHintBlock} isDark>
+    <PageLayout title={title} hint={resolved && (hintBlock || getHintBlock())} isDark>
```

Comment:
Why did you show the hint only when the extensions are resolved?

At least the title subline "Select a way to create an Application, component or service from one of the options." should be shown before that. At the moment the page jumps slightly down after the extensions are loaded. See gif:

![Peek 2021-05-20 16-30](https://user-images.githubusercontent.com/139310/118997202-c3681080-b988-11eb-8a21-ddfb7d1f8f4b.gif)